src/main/java/reactor/netty/resources/PooledConnectionProvider.java

```diff
@@ -86,7 +76,7 @@
 	final int                          maxConnections;
 
 	PooledConnectionProvider(String name, PoolFactory poolFactory) {
-		this(name, poolFactory, 0, -1);
+		this(name, poolFactory, 0, MAX_CONNECTIONS_ELASTIC);
 	}
```

Comment:
Just delete this constructor and use only the one with the builder (this constructor is used only in a test so adapt the test)